userguide/drawing_classifier/export-coreml.md

```diff
@@ -21,20 +21,14 @@ Making a prediction at inference time on device is easy! In this section, we go
 over your workflow depending on what input you may have in your app at inference
 time.
 
+
 #### Using Bitmap Input
 
 At inference time, if you have access to a bitmap and/or image that represents
 the drawing you want to classify, you can use the Vision framework to consume
 the exported Core ML model. Note that the image you provide to the Vision API
 must be a Grayscale Image. 
 
-```swift
-let model = try VNCoreMLModel(for: MySquareTriangleClassifier().model)
-
-let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
-    self?.processClassifications(for: request, error: error)
-})
-```
```

Comment:
After looking at the final result here, I think it's too easy for someone glancing at this page to miss the fact that content "Using Stroke-Based Drawing Input" applies to the bitmap case here. Let's just have two copies of the Vision sample code, which will at least differ in how the input image is obtained.